src/sentry/mail/adapter.py

```diff
@@ -108,10 +123,13 @@ def get_sendable_users(self, project):
 
     @staticmethod
     def notify(notification, target_type, target_identifier=None, **kwargs):
-        if target_type == ActionTargetType.RELEASE_MEMBERS:
-            ActiveReleaseAlertNotification(notification, target_type, target_identifier).send()
-        else:
-            AlertRuleNotification(notification, target_type, target_identifier).send()
+        AlertRuleNotification(notification, target_type, target_identifier).send()
```

Comment:
Undo hack where we look at the action target_type. No user-defined actions should have `target_type=RELEASE_MEMBERS` since that won't be exposed in the UI at all anymore.